taxi/forms.py

```diff
@@ -50,3 +50,30 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class DriverSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Search by username"}),
+    )
+
+
+class CarSearchForm(forms.Form):
+    model = forms.CharField(
+        max_length=255,
+        required=True,
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Search by car model"})
+    )
+
+
+class ManufacturerSearchForm(forms.Form):
+    name = forms.CharField(
+        max_length=255,
+        required=True,
```

Comment:
The 'required' attribute is set to True for the 'name' field in 'ManufacturerSearchForm'. This means the search form will not submit unless this field is filled, which might not be the intended behavior for a search form. Consider setting 'required' to False if you want to allow searches without specifying a manufacturer name.